tests/system/datastore/test_v3io.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shlex
+import subprocess
+
+import pytest
+
+import mlrun.datastore
+from tests.system.base import TestMLRunSystem
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestV3ioDataStore(TestMLRunSystem):
+    @staticmethod
+    def _skip_set_environment():
+        return True
+
+    def test_v3io_large_object(self):
+        import random
+
+        with open("tempfile_1", "wb") as f:
```

Comment:
Might want to consider using [tempfile](https://docs.python.org/3/library/tempfile.html). Maybe `mkdtemp()` since we also need to download the file back.